scripts/train/train.py

```diff
@@ -575,4 +19,4 @@ def main(cfg: DictConfig) -> Trainer:
     cli_cfg = om.from_cli(args_list)
     cfg = om.merge(yaml_cfg, cli_cfg)
     assert isinstance(cfg, DictConfig)
```

Comment:
Is the plan to deprecate this script? Shall we add a deprecation warning?

Reply:
I dont think we have current plan to deprecate, might be a follow up for daniel when he comes back